core/string/translation.h

```diff
@@ -77,6 +77,16 @@ class TranslationServer : public Object {
 
 	bool enabled = true;
 
+	bool pseudolocalization_enabled;
+	bool pseudolocalization_accents_enabled;
+	bool pseudolocalization_double_vowels_enabled;
+	bool pseudolocalization_fake_bidi_enabled;
+	bool pseudolocalization_override_enabled;
+	bool pseudolocalization_skip_placeholders_enabled;
+	float expansion_ratio;
```

Comment:
These should be given default values:
```suggestion
	bool pseudolocalization_enabled = false;
	bool pseudolocalization_accents_enabled = false;
	bool pseudolocalization_double_vowels_enabled = false;
	bool pseudolocalization_fake_bidi_enabled = false;
	bool pseudolocalization_override_enabled = false;
	bool pseudolocalization_skip_placeholders_enabled = false;
	float expansion_ratio = 0.0;
```